### PR TITLE
{devel}[dummy/dummy] libelf v0.8.13

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

I used this in a `--new-pr` demo, but we might as well merge it.

File: easybuild/easyconfigs/l/libelf/libelf-0.8.13.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+homepage = 'http://www.mr511.de/software/english.html'
+description = """libelf is a free ELF object file access library"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.mr511.de/software/']
+
+sanity_check_paths = {
+    'files': ['lib/libelf.a', 'lib/libelf.%s' % SHLIB_EXT, 'lib/libelf.so.0', 'include/libelf.h'],
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libelf/libelf-0.8.13.eb]: drop the `.so.0`?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8fa5938a2bdad11aac8c for a full test report.

**Comment by boegel** [comment]
```
Sanity check failed: no file of ('include/libelf.h',) in ...
```

O_o

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ef8d2d05669080c4f7ba for a full test report.

**Comment by migueldiascosta** [comment]
@boegel, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**